### PR TITLE
Asset sync 4.15

### PR DESCRIPTION
**Pull request by jonesbr17**
Sync assets with upstream

### CODE REVIEW

File: assets/cluster-bootstrap/cluster-network-01-crd.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # This is the advanced network configuration CRD
 # Only necessary if you need to tweak certain settings.
 # See https://github.com/openshift/cluster-network-operator#configuring
-# Source: https://github.com/openshift/cluster-network-operator/blob/release-4.14/manifests/0000_70_cluster-network-operator_01_crd.yaml
+# Source: https://github.com/openshift/cluster-network-operator/blob/release-4.15/manifests/0000_70_cluster-network-operator_01_crd.yaml
```
Comment (rtheis) [inline assets/cluster-bootstrap/cluster-network-01-crd.yaml:5]: This link is not valid.

File: assets/cluster-bootstrap/deployer-clusterrole.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Source: https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/release-4.14/bindata/v3.11.0/openshift-controller-manager/deployer-clusterrole.yaml
+# Source: https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/release-4.15/bindata/v3.11.0/openshift-controller-manager/deployer-clusterrole.yaml
```
Comment (rtheis) [inline assets/cluster-bootstrap/deployer-clusterrole.yaml:1]: This link is not valid either.

**Review by rtheis — CHANGES_REQUESTED** [review]
Please ensure all new links are valid.  Given parts have moved, we at least need to change the links but may also have functional changes to include.

**Comment by jonesbr17** [comment]
Fixed the invalid links

**Review by rtheis — APPROVED** [review]
/lgtm
/approved

**Pull request merged**